Library/Formula/emacs.rb

```diff
@@ -43,6 +43,8 @@ class Emacs < Formula
   depends_on "mailutils" => :optional
   depends_on "glib" => :optional
 
+  patch :DATA
```

Comment:
Do you mind adding a comment about what this does?

Reply:
Good call, will do 👍 !